### PR TITLE
Adding translation to rule editor and transaction table

### PR DESCRIPTION
**Pull request by manuelcanepa**
This PR will add translation to Rule Editor and Simple Transaction Table.

### CODE REVIEW

**Comment by manuelcanepa** [comment]
I need some help to check (And translate) this part of the file: 

https://github.com/actualbudget/actual/blob/i18n/packages/desktop-client/src/components/modals/EditRule.js#L240-L265

File: packages/desktop-client/src/components/accounts/Account.js [hunk 1]
```diff
@@ -303,8 +308,33 @@ function DetailedBalance({ name, balance }) {
...
+      )}
+      {name === 'uncleared' && (
+        <Trans
+          values={{
+            amount: format(balance, 'financial')
+          }}
+        >
+          {'account.unclearedTotal'}
+        </Trans>
+      )}
```
[the remark this answers is not in the document]
Comment (TomAFrench) [reply packages/desktop-client/src/components/accounts/Account.js:337]: Or even just pass in the translation keys as `name` directly

[the remark this answers is not in the document]
Comment (TomAFrench) [reply packages/desktop-client/src/components/accounts/Account.js:337]: Looks like we're supposed to enumerate the potential values in comments in that case?

https://github.com/i18next/i18next-parser#caveats

[the remark this answers is not in the document]
Comment (manuelcanepa) [reply packages/desktop-client/src/components/accounts/Account.js:337]: The workaround of using comments is a good one too.

[the remark this answers is not in the document]
Comment (manuelcanepa) [reply packages/desktop-client/src/components/accounts/Account.js]: @TomAFrench tbh, i like that workaround

File: packages/desktop-client/src/locales/es-ES.json [hunk 2]
```diff
@@ -1,59 +1,151 @@
...
+    "passwordCannotBeEmpty": "La contraseña no puede estar vacía",
+    "passwordsDoNotMatch": "Las contraseñas no coinciden",
+    "passwordSuccessfullyChanged": "La contraseña fue modificada exitosamente",
     "setPassword": "Establecer una contraseña para esta instancia de servidor",
+    "thisWillChangeThePasswordAdvice": "Este proceso modificará la contraseña para el servidor. Todas las sesiones existentes permanecerán conectadas.",
     "title": "Bootstrap esta instancia de Actual",
-    "tryDemo": "Probar Demo"
+    "tryDemo": "Probar Demo",
+    "unableToContactTheServer": "Incapaz de conectarse con el servidor",
+    "whoopsAnErrorOccurredOnOurSide": "¡Vaya, ocurrió un error de nuestro lado! Intentaremos solucionarlo pronto."
```
Comment (j-f1) [inline packages/desktop-client/src/locales/es-ES.json:41]: Suggestion:

```suggestion
    "unknownError": "¡Vaya, ocurrió un error de nuestro lado! Intentaremos solucionarlo pronto."
```

**Pull request merged**